deploy/install/01-prerequisite/04-default-setting.yaml

```diff
@@ -44,3 +44,5 @@ data:
     #kubernetes-cluster-autoscaler-enabled:
     #orphan-auto-deletion:
     #storage-network:
+    #failed-backup-auto-deletion:
+    #failed-backup-ttl:
```

Comment:
Is it possible to have just one setting `failed-backup-ttl` instead of two settings? 
For example,
if failed-backup-ttl == 0, disable the auto-deletion. 
if failed-backup-ttl > 0, enable the auto-deletion.

Reply:
Yes, use only `failed-backup-ttl` and disabled if `-1`, no wait deleting if `0`, wait for x seconds if x > 0. Thanks, that's more simple and clear.

Reply:
Changed, disable if failed-backup-ttl == 0, enable if failed-backup-ttl > 0 for consistency usage.